students/11th/hyojeung/main.html

```diff
@@ -0,0 +1,183 @@
+              <div><img alt="logo" src="img/logoicon_nav.png" width="40"/></div>
+              <div class="logoline"></div>
+              <div class="logoimg"><a href="main.html"><img alt="logo" src="img/logo_nav.png" /></a></div>
+            </ul>
+          </div>
+
+          <div class="searchbox">
+            <input type="text" id="search" placeholder="검색" />
+          </div>
+
+          <div class="gnbbox">
+            <div class="gnbicon"><img alt="explore" src="img/icon2_nav.png"/></div>
+            <div class="gnbicon"><img alt="like" src="img/icon3_nav.png"/></div>
+            <div class="gnbicon"><img alt="profile" src="img/icon4_nav.png"/></div>
+          </div>
+        </nav>
+        <!-- 메인 -->
+        <div class="mainbox">
+          <article class="articlebox">
+            <div class="articletop">
+                  <div class="fomproimg"></div>
+                  <span class="forname">chyapchyap_cake</span>
+                  <div class="minimenu"><img alt="menu" src="img/menu_1.png"/></div>
+            </div>
+
+                <div class="middleimg">
+                  <img src="img/fomimg.jpg" width="612"/>
+                </div>
+
+                <div class="middleicon">
+                  <div class="icon1" onclick="changehHeart()"><img id="heartImg" src="img/icon1_fom.png"/></div>
+                  <div class="icon1"><img alt="texticon" src="img/icon2_fom.png" /></div>
+                  <div class="icon1"><img alt="directicon" src="img/icon3_fom.png" /></div>
+                  <div class="icon1"><img alt="keepicon" src="img/icon4_fom.png" /></div>
+                </div>
+
+                <div class="middletext">
+                  <div class="boldtext">
+                    <div class="boldtextimg"></div>
+                    <b>aineworld</b>님 외 <b>10</b>명이 좋아합니다
+                  </div>
+                  <ul class="maintext marginbt">
+                    <li class="namebold">canon_mj</li>
+                    <li id="moretext">보틀케이크 큰사이즈2개,작은사이즈15개 나왔어요✨
+예약판매는 어렵습니다 선착순으로 판매할게요 현재 작업이 많아 주문케이크 상담 답장 못 드리고 있어요 작업이 끝난 저녁쯤 순차적으로 답장 드릴게요🙏</li>
+                    <li id="morebtn">더 보기</li>
+                  </ul>
+
+                </div>
+                <span class="timetext">42분 전</span>
+                <div class="bottom">
+                  <input type="text" id="commentinput" placeholder="댓글 달기...."/>
+                  <input type ="button" id="commentbtn" value="게시" />
+                </div>
+          </article>
+        <!-- article완료 / aside시작  -->
+        <aside class="asidebox">
+          <div class="profile">
+              <div class="profileimg"></div>
+            <ul class="profiletext">
+              <li class="profileid"><a href="#">ovv__0703</a></li>
+              <li class="profilename">ㅎㅈ</li>
+            </ul>
+          </div>
+
+          <div class="storeybox">
+            <ul class="storeyboxtop">
+              <li class="storeyboxtitle">스토리</li>
```

Comment:
```suggestion
          <div class="storyBox">
            <ul class="storyBoxTop">
              <li class="storyBoxTitle">스토리</li>
```
1. 오타 수정해주세요
2. camel case 사용해주세요

Reply:
1. 오타수정완료하였습니다.
2. camel case로 모든 id, class명 수정하였습니다.